spec/javascripts/app/webauthn_spec.js

```diff
@@ -41,15 +40,15 @@ describe('WebAuthn', () => {
     const userId = '123';
     const userEmail = 'test@test.com';
     const userChallenge = '[1, 2, 3, 4, 5, 6, 7, 8]';
-    const excludeCredentials = 'credential123,credential456';
+    const excludeCredentials = 'Y3JlZGVudGlhbDEyMw==,Y3JlZGVudGlhbDQ1Ng=='; // Base64-encoded 'credential123,credential456'
 
     it('enrolls a device using the proper create options', (done) => {
       const expectedCreateOptions = {
         publicKey: {
-          challenge: Buffer.from([1, 2, 3, 4, 5, 6, 7, 8]),
-          rp: { name: 'testing.webauthn.js' },
+          challenge: new Uint8Array([1, 2, 3, 4, 5, 6, 7, 8]),
+          rp: { name: 'example.test' },
```

Comment:
qq: why did `rp.name` need to change here?

Reply:
> qq: why did `rp.name` need to change here?

This is from how the `window` global is stubbed.

In master:

https://github.com/18F/identity-idp/blob/f6cd1c3b1371eefaf592a472eecbc245b0bed059/spec/javascripts/app/webauthn_spec.js#L7-L11

After the changes, it will read from the hostname of the JSDOM global:

https://github.com/18F/identity-idp/blob/3a63a7c4a199a6f52e37dec5e199473f8356e970/spec/javascripts/support/dom.js#L8-L10